ang/civicase/case/details/people-tab/directives/case-details-people-tab.directive.html

```diff
@@ -26,11 +26,11 @@
           </li>
```

Comment:
Thanks @reneolivo . But can we refactor this a little bit more, by breaking the HTML into smaller chunks. No need to create directives for now, but will be great if we can use ng-include and divide the file into Header, Footer, Filter etc.